assets/javascripts/kitten/karl/pages/kissbankers-filters/components/annexe-nav.js

```diff
@@ -0,0 +1,77 @@
+import React, { PureComponent } from 'react'
+import styled, { css } from 'styled-components'
+import { Button, ICON } from '../../../../components/buttons/button/button'
+import { TextInput } from '../../../../components/form/text-input'
+import { ExportIcon } from '../../../../components/icons/export-icon'
+import { EmailIcon } from '../../../../components/icons/email-icon'
+import { ScreenConfig } from '../../../../constants/screen-config'
+import { pxToRem } from '../../../../helpers/utils/typography'
+
+const StyledButtonContainer = styled.div`
+  display: flex;
+  margin-bottom: ${pxToRem(15)};
+
+  @media (min-width: ${ScreenConfig.S.min}px) {
+    margin-bottom: ${pxToRem(20)};
+  }
+`
+
+const buttonMargin = css`
+  @media (max-width: ${ScreenConfig.M.max}px) {
+    ${ICON}
+
+    > : nth-child(n) {
```

Comment:
```suggestion
    > :nth-child(n) {
```

à quoi sert ce sélecteur CSS ? le `:nth-child(n)` sélecte tous les enfants non ?